packages/editor/src/editor/sidebar/fields/datatable-columns/useDataTableColumns.ts

```diff
@@ -72,7 +72,9 @@ const convertBrowserNodesToColumns = (nodes: Array<BrowserNode<Variable>>): Data
         id: createId('DataTableColumn'),
         config: {
           header: node.data?.attribute ?? '',
-          value: ''
+          value: '',
+          filterable: false,
+          sortable: false
         }
```

Comment:
Sorry only saw this now...
The problem that I see here is that we now duplicate default values. So if we would like to change this ever have to change them at 3 places.
Instead, you could simply reuse the defaultProps object from the DataTableColumn config. Or even better just use the create method on the config object, because that exactly the usecase I've build it for ;)